tests/driver_srf04/README.md

```diff
@@ -0,0 +1,14 @@
+tests/driver_srf04
+================
+This example shows the usage of an srf04 module.
+The application exerts a timer and two gpio.
+The module is an ultrasonic range finder with a small protocoll,
+hence an implementation can be kept simple.
```

Comment:
I don't think this comment is very useful. Better detail the mentioned protocol ? or give a link to a datasheet.